.github/CODEOWNERS

```diff
@@ -30,175 +30,257 @@
 /sdk/ @xirzec
 
 # PRLabel: %Azure.Core
```

Comment:
@jsquire since triage is being turned on at the end of the week, am I okay to merge this PR as-is and we can further refine as we learn more about how to specify labels that belong to the SDK team and not a service team?

Reply:
I don't see anything blocking beyond the permissions issue that would need to be added to the baseline errors.   Most of this will just consider you the "Service Attention" peoples.